### ISSUE TITLE
NotImplementedException on ParseRoleStore

### ISSUE THREAD

**Issue by elpoul** [corrego/Parse.AspNet.Identity#5]
Hi
I've tried to use your library on a Parse solution and I've noticed that the **ParseRoleStore** class throws a NotImplemented on dispose which could result in unwanted behavior. Was there a specific reason for implementing it like that?

Cheers

**Comment by corrego** [comment]
Not at all, I implemented the bare-bones functionality and not much else... everything else is throwing not implemented by default. I wasn't paying attention. I will look around soon and implement proper disposal. Thanks for bringing this up.
